apps/client-server/src/app/form-generator/form-generator.service.spec.ts

```diff
@@ -127,16 +126,15 @@ describe('FormGeneratorService', () => {
           tags: [],
         },
         formField: 'tag',
-        i18nLabel: 'form.tags',
-        label: 'Tags',
+        label: 'tags',
```

Comment:
q: Why are you lowercasing these and many others?

Reply:
because its now i18nLabel actually